lib/source-map/source-map-consumer.js

```diff
@@ -199,10 +198,14 @@ define(function (require, exports, module) {
                                     this._originalMappings,
                                     "originalLine",
                                     "originalColumn",
-                                    util.compareByOriginalPositions);
+                                    util.compareByOriginalPositions,
+                                    true);
```

Comment:
I don't think this `true` needs to be passed here. Left over from earlier iteration? Confirm and I can merge and remove it.